pom.xml

```diff
@@ -62,7 +62,7 @@
         <gravitee-fetcher-api.version>1.4.0</gravitee-fetcher-api.version>
         <gravitee-gateway-api.version>2.1.0-alpha.12</gravitee-gateway-api.version>
         <gravitee-license-node.version>1.3.1</gravitee-license-node.version>
-        <gravitee-node.version>3.0.0-alpha.1</gravitee-node.version>
+        <gravitee-node.version>2.0.4</gravitee-node.version>
```

Comment:
Not sure we should "downgrade" this 🤔 I thought it was related to vertx upgrade 🤔 
If 2.0.4 brings a fix, it should be applied on the `alpha` branch of `gravitee-node`

Reply:
This was also question for myself but I have no idea where this version 3.0.0-alpha.1 comes from? The latest version in gravitee-node master branch was 2.0.3

Reply:
As I said, the "alpha" version comes from the `alpha` branch: https://github.com/gravitee-io/gravitee-node/tree/alpha

Reply:
Now it should be OK